xdcs-frontend/src/app/element/log-preview/log-preview.component.less

```diff
@@ -20,6 +20,12 @@ pre.log-output {
     filter: brightness(50%);
   }
 
+  .line-tag {
+    margin-right: 5px;
+    text-align: right;
+    min-width: 4ch;
```

Comment:
4 to chyba za mało, 16 wydaje się rozsądne ale nie wiem czy nie wyglądałoby to słabo
chyba trzeba zrbić mechanizm, który dostosowuje szerokość